src/index.js

```diff
@@ -1,203 +1,83 @@
-      instance: Element,
-      preparedUpdateQueue: Array<[string, any]>
-    ): void {
-      DOMComponent.updateProps(instance, preparedUpdateQueue);
-    },
+  let root = new Root(domContainer, DOMLiteReconciler);
+  ContainerMap.set(domContainer, root);
+  // Initial render only is unbatched
+  return DOMLiteReconciler.unbatchedUpdates(() =>
+    root.render(elements, callback)
+  );
+}
 
-    commitMount() {
-      // noop
-    },
+function unmountComponentAtNode(domContainer: DOMContainer): boolean {
+  invariant(
+    domContainer && [1, 8, 9, 11].indexOf(domContainer.nodeType) !== -1,
+    'unmountComponentAtNode(...): Target container is not a DOM element.'
+  );
 
-    commitTextUpdate(textInstance: Text, oldText: string, newText: string) {
-      textInstance.nodeValue = newText;
-    },
+  const root = ContainerMap.get(domContainer);
 
-    resetTextContent(domElement: Element): void {
-      domElement.textContent = '';
-    },
+  if (!root) return false;
 
-    appendChild(parentInstance: Element, child: Element | Text): void {
-      parentInstance.appendChild(child);
-    },
+  DOMLiteReconciler.unbatchedUpdates(() => {
+    root.unmount(() => {
+      ContainerMap.delete(domContainer);
+    });
+  });
 
-    appendChildToContainer(
-      parentInstance: DOMContainer,
-      child: Element | Text
-    ): void {
-      parentInstance.appendChild(child);
-    },
-    insertBefore(
-      parentInstance: Element,
-      child: Element | Text,
-      beforeChild: Element | Text
-    ): void {
-      parentInstance.insertBefore(child, beforeChild);
-    },
+  return true;
+}
 
-    insertInContainerBefore(
-      container: Element,
-      child: Element | Text,
-      beforeChild: Element | Text
-    ): void {
-      container.insertBefore(child, beforeChild);
-    },
+function findDOMNode(
+  componentOrElement: Element | ?React$Component<any, any>
+): null | Element | Text {
+  if (componentOrElement == null) return null;
 
-    removeChild(parentInstance: Element, child: Element | Text): void {
-      parentInstance.removeChild(child);
-    },
-    removeChildFromContainer(
-      parentInstance: DOMContainer,
-      child: Element | Text
-    ): void {
-      parentInstance.removeChild(child);
-    }
+  if (componentOrElement.nodeType === 1 || componentOrElement === 3) {
+    return (componentOrElement: any);
   }
-};
-
-const DOMLiteRenderer = Reconciler(hostConfig);
-
-DOMLiteRenderer.injectIntoDevTools({
-  bundleType: 1,
-  version: '0.1.0',
-  rendererPackageName: 'react-dom-lite',
-  findFiberByHostInstance: getInternalHandleFromInstance
-});
-
-let ContainerMap = new WeakMap();
-function render(elements: React$Element<any>, domContainer: Element) {
-  const container = DOMLiteRenderer.createContainer(domContainer, false, false);
-  const root = new Root(container, DOMLiteRenderer);
-
-  ContainerMap.set(domContainer, root);
+  // $FlowFixMe
+  const handler: OpaqueHandle = componentOrElement._reactInternalFiber;
```

Comment:
I don't see another way to handle this without touching the internal bits here?

Reply:
Yeah, doesn't seem like it's exposed right now.